src/utilities/include/public.h

```diff
@@ -598,7 +599,6 @@ class Quality {
  *  @brief Custom Exceptions
  * @{
  */
-#ifdef MANIFOLD_DEBUG
```

Comment:
@pca006132 Is this okay to remove? I thought this was required to make us able to compile with exceptions disabled?

Reply:
It is fine, the header files are still there even when we use `-fno-exceptions`. We just cannot throw them. I tried compiling it and it is fine.